manifests/init.pp

```diff
@@ -40,6 +40,11 @@
   $disable_monitor     = false,
 ) {
 
+  # validate type as array or fail
+  if is_array($servers) != true {
```

Comment:
This would be simpler with just `validate_array()`. Also need corresponding spec tests to show that the failure occurs if you pass a non-array.